x-pack/plugin/esql/qa/testFixtures/src/main/resources/enrich.csv-spec

```diff
@@ -88,18 +88,18 @@ ROW left = "left", client_ip = "172.21.0.5", env = "env", right = "right"
 left:keyword | env:keyword | right:keyword | client_ip:keyword
 ;
 
-shadowingMulti
+shadowingMulti#[skip:-8.13.99, reason:ENRICH extended in 8.14.0]
```

Comment:
Could you use a required-capability instead? I think that's the "modern" way to do these"

Reply:
I don't like this either, but the skip is in line with the skip for `spatialEnrichmentKeywordMatch` (this test uses the same enrich command and policy).
The PR that introduced this `shadowingMulti` test didn't enable this behavior or anything, it just provided more coverage.

We _could_ solve this skip (and the other skips in this file) with a capability, but we'd need to backport this to 8.14 lest we lose coverage for perfectly valid multi-node/cluster tests; because of this, I think this should be a separate PR, if we want that.